### PR TITLE
Extract

### PR DESCRIPTION
**Pull request by tjquinn1**

### CODE REVIEW

File: MAXQDA-Conversion/req.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pandas=0.23.3=py36h6440ff4_0
+numpy=1.14.5=py36h648b28d_4
+numpy-base=1.14.5=py36ha9ae307_4
```
Comment (jdamerow) [inline MAXQDA-Conversion/req.txt]: I get error messages when I try to install this:
```Invalid requirement: 'pandas=0.23.3=py36h6440ff4_0'```

Also, let's call the file "requirements.txt".

**Review by jdamerow — DISMISSED** [review]

**Comment by jdamerow** [comment]
Invalid requirement: 'pandas=0.23.3'
= is not a valid operator. Did you mean == ?

**Comment by jdamerow** [comment]
pathlib is still missing

File: MAXQDA-Conversion/Extract.ipynb [hunk 2]
```diff
@@ -12,7 +12,19 @@
...
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
+      "\u001b[0;32m<ipython-input-1-00e8912423f8>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0murllib\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 8\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mpathlib\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mPath\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      9\u001b[0m \u001b[0;31m# Uncomment the next two lines if you want to see expanded output\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     10\u001b[0m \u001b[0;31m#pd.options.display.max_colwidth\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mImportError\u001b[0m: No module named pathlib"
+     ]
+    }
+   ],
```
Comment (jdamerow) [inline MAXQDA-Conversion/Extract.ipynb]: let's remove this error message

File: MAXQDA-Conversion/Extract.ipynb [hunk 3]
```diff
@@ -145,56 +163,66 @@
...
    "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
```
Comment (jdamerow) [inline MAXQDA-Conversion/Extract.ipynb]: ?

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: MAXQDA-Conversion/Extract.ipynb (hunk 3 above)
Comment (tjquinn1) [reply MAXQDA-Conversion/Extract.ipynb]: I am not sure why it changed it to python2 on my machine it says python3
screen shot 2018-10-09 at 11 17 18 am

**Comment by jdamerow** [comment]
ok cool. last thing: there needs to be a blank after the heading symbols ('###'), otherwise the heading isn't rendered.
image
vs.
image

**Pull request merged**